music_assistant/controllers/players/player_controller.py

```diff
@@ -102,23 +103,24 @@ def handle_player_command[PlayerControllerT: "PlayerController", **P, R](
     @functools.wraps(func)
     async def wrapper(self: PlayerControllerT, *args: P.args, **kwargs: P.kwargs) -> R | None:
```

Comment:
```suggestion
    async def wrapper(self: PlayerControllerT, *args: P.args, **kwargs: P.kwargs) -> None:
```
The return type can actually be None because all of the player "cmd_" methods are not returning anything